### ISSUE TITLE
关于训练中计算损失的问题

### ISSUE THREAD

**Issue by zhaoxjmail** [shibing624/pycorrector#420]
### Describe the Question
在下图中显示计算损失时（w*loss_det+(1-w)*loss_cor）与代码中不一致（代码中outputs[1]应该是loss_cor，outputs[0]是loss_det），请问哪一个是正确的？谢谢您！
 
![image](https://github.com/shibing624/pycorrector/assets/6505251/77fc7b0a-9b48-4530-802d-8f954e9730e6)

代码中base_model.py 第127，128行：
outputs = self.forward(ori_text, cor_text, det_labels)
loss = self.w * outputs[1] + (1 - self.w) * outputs[0]

**Comment by shibing624** [comment]
都是正确的，w是设置的。无非设置0.3和0.7的区别而已。
